canonical/snapcraft.io
# feat: Move publicise into publisher app

## Done
Moved the publicise page into the publisher app using the application layout

## How to QA
- Go to https://snapcraft-io-4844.demos.haus/steve-test-snap/publicise
- Check that is has the "Publicise" view and that it works like production, e.g. https://snapcraft.io/steve-test-snap/publicise
- Go to https://snapcraft-io-4844.demos.haus/teststeve1/publicise and check that there is a notification and the page is disabled
- Go to https://snapcraft-io-4844.demos.haus/teststeve1/publicise/badges
- Check that it is a 404
- Go to https://snapcraft-io-4844.demos.haus/teststeve1/publicise/cards
- Check that it is a 404

## Testing

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-14631
